src/otoole/write_strategies.py

```diff
@@ -152,8 +152,7 @@ def _write_parameter(
         default : int
         """
 
-        if not self.write_defaults:
-            df = self._form_parameter(df, default)
+        df = self._form_parameter(df, default)
```

Comment:
Here is where datafile defaults are being filtered out, even if `write_defaults` is passed. The issue is that `write_defaults` is no longer exposed to the write strategy, so we need a way to address this conditional.